### PR TITLE
docs(telemetry): add docs about standard instruments for subgraph

### PR DESCRIPTION
**Pull request by bnjjj**
Added documentations about standard instruments available at the subgraph service:

  * `http.client.request.body.size` - A histogram of request body sizes for requests handled by subgraphs.
  * `http.client.request.duration` - A histogram of request durations for requests handled by subgraphs.
  * `http.client.response.body.size` - A histogram of response body sizes for requests handled by subgraphs.

These instruments are configurable in `router.yaml`:

```yaml title="router.yaml"
telemetry:
  instrumentation:
    instruments:
      subgraph:
        http.client.request.body.size: true # (default false)
        http.client.request.duration: true # (default false)
        http.client.response.body.size: true # (default false)
```

---

**Checklist**

Complete the checklist (and note appropriate exceptions) before the PR is marked ready-for-review.

- [ ] Changes are compatible[^1]
- [x] Documentation[^2] completed
- [ ] Performance impact assessed and acceptable
- Tests added and passing[^3]
    - [ ] Unit Tests
    - [ ] Integration Tests
    - [ ] Manual Tests

**Exceptions**

*Note any exceptions here*

**Notes**

[^1]: It may be appropriate to bring upcoming changes to the attention of other (impacted) groups. Please endeavour to do this before seeking PR approval. The mechanism for doing this will vary considerably, so use your judgement as to how and when to do this.
[^2]: Configuration is an important part of many changes. Where applicable please try to document configuration examples.
[^3]: Tick whichever testing boxes are applicable. If you are adding Manual Tests, please document the manual testing (extensively) in the Exceptions.

### CODE REVIEW

File: .changesets/docs_bnjjj_docs_standard_instr.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+### docs(telemetry): add docs about standard instruments for subgraph ([PR #5422](https://github.com/apollographql/router/pull/5422))
```
Comment (abernix) [inline .changesets/docs_bnjjj_docs_standard_instr.md:1]: As it'll be in the Documentation section, I think it'll be more presentable to just say something like:

```suggestion
### Standard instrument configuration documentation for subgraphs ([PR #5422](https://github.com/apollographql/router/pull/5422))
```

**Review by abernix — COMMENTED** [review]
Drive by comment.  Would be good for @BrynCooke and @Meschreiber to take a looksie beyond that.

File: .changesets/docs_bnjjj_docs_standard_instr.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+### Standard instrument configuration documentation for subgraphs ([PR #5422](https://github.com/apollographql/router/pull/5422))
+
+Added documentations about standard instruments available at the subgraph service:
```
Comment (Meschreiber) [inline .changesets/docs_bnjjj_docs_standard_instr.md:3]:
```suggestion
Added documentation about standard instruments available at the subgraph service level:
```

Comment (Meschreiber) [reply .changesets/docs_bnjjj_docs_standard_instr.md:3]: (I believe we've historically used "at" for talking about a "service level" ([e.g.](https://github.com/apollographql/router/blob/dev/CHANGELOG.md#-documentation-8)) and "in" when talking about something happening during or in a service. ([e.g.](https://www.apollographql.com/docs/router/configuration/telemetry/instrumentation/instruments/#value)))

File: docs/source/configuration/telemetry/instrumentation/instruments.mdx [hunk 3]
```diff
@@ -17,10 +17,19 @@ You can configure instruments in `router.yaml` with `telemetry.instrumentation.i
 
 OpenTelemetry specifies multiple [standard metric instruments](https://opentelemetry.io/docs/specs/semconv/http/http-metrics/) that are available in the router:
 
-* `http.server.active_requests` - The number of active requests in flight.
-* `http.server.request.body.size` - A histogram of request body sizes for requests handled by the router.
-* `http.server.request.duration` - A histogram of request durations for requests handled by the router.
-* `http.server.response.body.size` - A histogram of response body sizes for requests handled by the router.
++ At the router service:
```
Comment (Meschreiber) [inline docs/source/configuration/telemetry/instrumentation/instruments.mdx:20]:
```suggestion
+ In the [router service](#router-request-lifecycle-services):
```

File: docs/source/configuration/telemetry/instrumentation/instruments.mdx [hunk 4]
```diff
@@ -17,10 +17,19 @@ You can configure instruments in `router.yaml` with `telemetry.instrumentation.i
...
+
+  * `http.server.active_requests` - The number of active requests in flight.
+  * `http.server.request.body.size` - A histogram of request body sizes for requests handled by the router.
+  * `http.server.request.duration` - A histogram of request durations for requests handled by the router.
+  * `http.server.response.body.size` - A histogram of response body sizes for requests handled by the router.
+
++ At the subgraph service:
```
Comment (Meschreiber) [inline docs/source/configuration/telemetry/instrumentation/instruments.mdx:27]:
```suggestion
+ In the [subgraph service](#router-request-lifecycle-services):
```

**Review by Meschreiber — APPROVED** [review]
Left some preposition nits, otherwise LGTM!

**Pull request merged**